core/App/components/misc/CredentialCard11.tsx

```diff
@@ -272,8 +407,16 @@ const CredentialCard11: React.FC<CredentialCard11Props> = ({ credential, style =
 
   const CredentialCardSecondaryBody: React.FC = () => {
     return (
-      <View testID={testIdWithKey('CredentialCardSecondaryBody')} style={styles.secondaryBodyContainer}>
-        {overlay.cardLayoutOverlay?.backgroundImageSlice?.src ? (
+      <View
+        testID={testIdWithKey('CredentialCardSecondaryBody')}
+        style={[
+          styles.secondaryBodyContainer,
+          {
+            backgroundColor: !error ? styles.secondaryBodyContainer.backgroundColor : ColorPallet.notification.error,
```

Comment:
Using negative logic like this can lead to errors. Simple enough to just say: "if there's an error, show the error otherwise show the default"